3-box-model/styles.css

```diff
@@ -0,0 +1,32 @@
+*{
```

Comment:
```suggestion
body {
```
Avoid the use of  `*` it affects performance.